build/config.props

```diff
@@ -37,7 +37,7 @@
 
     <!-- .NET Core SDK Insertion Logic -->    
     <CliBranchForTesting Condition="'$(OverrideCliBranchForTesting)' != ''">$(OverrideCliBranchForTesting)</CliBranchForTesting>
-    <CliBranchForTesting Condition="'$(OverrideCliBranchForTesting)' == ''">release/3.1.2xx</CliBranchForTesting>
+    <CliBranchForTesting Condition="'$(OverrideCliBranchForTesting)' == ''">3.1</CliBranchForTesting>
```

Comment:
As per [doc](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#options), The default value of channel is `LTS`. I have modified the `common.ps1` locally to accept default value for `channel` parameter and executed `configure.ps1` which has completed successfully. I think the default value will help us avoid hard-coding the channel version. Thoughts ?

Reply:
LTS could change over time. Are we going to always test in LTS or in 2.2 ?

Reply:
We should be testing with the version that's closest to what we insert into, because when there's a new feature that we depend on, we won't be able to use anything lower. Consider for example when Rob finishes his `dotnet nuget sources` work. He can't create tests for the .NET CLI, until the .NET CLI has those commands, but once it does and tests are added, we will no longer be able to test with any lower version of the .NET CLI.

So, `Current` would be a better default than `LTS`, if we don't want to update this version ever again, but then we won't be able to add tests for anything that uses new features until it goes GA.